PROJECT

```diff
@@ -2,30 +2,62 @@ domain: hashicorp.com
 layout: go.kubebuilder.io/v2
 repo: github.com/hashicorp/consul-k8s
 resources:
```

Comment:
Changes in this file are side-effects of updating to Operator SDK 1.15
It changes the version from `3-alpha` to `3` are requires additional fields per CRD